### PR TITLE
Feature/dcaballero/gs labels change

### PR DESCRIPTION
**Pull request by dcaballeroSf**
# Critical Changes

# Changes
Changing labels in Adoption Get Started page
# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by dcaballeroSf** [comment]
[WI-8532798](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008kiuRIAQ/view)

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -7149,7 +7149,7 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Link label of checklist item Running Reports in Get Started</shortDescription>
-        <value>Learn how to customize NPSP reports &amp; dashboards in Trailhead</value>
+        <value>Learn how to customize NPSP reports &amp; Dashboard in Trailhead</value>
```
Comment (force2b) [inline src/labels/CustomLabels.labels:7152]:
```suggestion
        <value>Learn how to customize NPSP reports &amp; Dashboards in Trailhead</value>
```

File: src/labels/CustomLabels.labels [hunk 2]
```diff
@@ -13800,15 +13808,15 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
     </labels>
     <labels>
         <fullName>gsResourcesItemProgramManagementLinkLabel</fullName>
-        <categories>Get Started Page</categories>
+        <categories>tart Managing Programs</categories>
```
Comment (force2b) [inline src/labels/CustomLabels.labels:13811]:
```suggestion
        <categories>Start Managing Programs</categories>
```

File: src/labels/CustomLabels.labels [hunk 3]
```diff
@@ -13800,15 +13808,15 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
...
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Resource Program Management link label in Get Started Resources section</shortDescription>
         <value>Get Started</value>
     </labels>
     <labels>
         <fullName>gsResourcesItemMarketingEngagementLabel</fullName>
-        <categories>Get Started Page</categories>
+        <categories>Start Engaging Constituents</categories>
```
Comment (force2b) [inline src/labels/CustomLabels.labels:13819]: Why did the category change here and below? That should be the same for all GetStarted labels

File: src/labels/CustomLabels.labels [hunk 4]
```diff
@@ -13872,10 +13880,10 @@ Note: The NPSP Settings tab is visible in the Nonprofit Success Pack application
     </labels>
     <labels>
         <fullName>gsVideoHeaderTitle</fullName>
-        <categories>Get Started Page</categories>
+        <categories>Get Started on NPSP</categories>
```
Comment (force2b) [inline src/labels/CustomLabels.labels:13883]:
```suggestion
        <categories>Get Started Page</categories>
```

**Review by force2b — CHANGES_REQUESTED** [review]

**Review by force2b — APPROVED** [review]

**Pull request merged**